country-info.csv

```diff
@@ -165,6 +165,7 @@ National Taiwan University,asia,tw
 National Tsing Hua University,asia,tw
 National University of Singapore,asia,sg
 Newcastle University,europe,uk
+NISER,asia,in

```

Comment:
Appears to be missing at least one faculty member (https://www.niser.ac.in/scps/faculty-list).

Reply:
I have updated now.